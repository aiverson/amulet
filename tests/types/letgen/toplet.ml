let id x = x
let main =
  let _ = id 1
  id ()
