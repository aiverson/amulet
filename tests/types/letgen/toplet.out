id : Infer{'a : type}. 'a -> 'a
main : unit
